README.md

```diff
@@ -1,16 +1,19 @@
 # Simulate 
 [![Gitter](https://badges.gitter.im/codEd-org/simulate.svg)](https://gitter.im/codEd-org/simulate?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-Collection of simulations and visualizations in a cross platform app made using Flutter.
+Welcome to Simulate! Simulate is a collection of simulations and visualizations from various domains like mathematics, physics, computer science etc. in a cross platform app made using Flutter. The outcomes of these simulations mostly are really smooth (thanks to Flutter) and extremely beautiful (thanks to mathematics/sciences behind it). You can take a peek here.
+New to Simulate? Search for more information here https://cod-ed.github.io/simulate/
 
 # Contributing
 
-Before contributing, we highly recommend introducing yourself and your ideas on [gitter](https://gitter.im/codEd-org/simulate).
+Before contributing, we highly recommend introducing yourself and your ideas on [gitter](https://gitter.im/codEd-org/simulate). You all can solve our [issues](https://github.com/cod-ed/simulate/issues) or help us find new to make this application more interesting and smooth.
```

Comment:
I think mentioning a link to issues is bit obsolete in a README, though it can help on the website.

Reply:
Edited as per requested @yashlamba